datasets/international.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
```

Comment:
These are done automatically by the jekyll-redirects plugin when a user tries to access the page. They don't need to be submitted to Github.

Reply:
In fact, should NOT be submitted to Github. :-)

Reply:
Im going to do this tomorrow because 1. it's getting late here :) and 2. it's really annoying to do this remotely (im wfh today). It ll be so much easier in person connecting to my machine.

And got it so we should add these to the .gitignore!

Reply:
Done.